arch/arc/core/isr_wrapper.S

```diff
@@ -396,39 +387,10 @@ irq_hint_handled:
 	ld_s r1, [r0, 4] /* ISR into r1 */
 #ifdef CONFIG_EXECUTION_BENCHMARKING
 	push_s r0
-	push_s blink
```

Comment:
for the isr start, blink was saved as well. Not needed here? Or not needed in the earlier code for isr start?

Reply:
no need to save here. Blink is safe after irq_hint_handled. Also just need to save r0 and r1